### PR TITLE
Implemented timezone conversion

### LINKED ISSUE #23: Timezone Converter
# Description

Timezone converter for the bot so that we can see what time are Weekly Syncs in our own timezones
# Acceptance Criteria

- Bot can send a list of timezones with their equivalent times given an input e.g 
1/9/21 1900 PDT ->
GMT +8: 2/9/21 1000
(this template can be changed)

### PR DESCRIPTION
**Pull request by jyorien**
# Description

- Added pytz in requirements.txt
- Commands added:
  - !time convert [dd/mm/yyyy] [hh:mm] [timezone]
  - !time add_timezone [timezone]
  - !time delete_timezone [timezone]
  - !time list_timezones

### CODE REVIEW

**Comment by pk400** [comment]
I like the new `TimeCog`!

I wonder what's up with the git history, its showing other changes in `main` unrelated to this PR, which is weird cause you already merged `main` here :thinking:. I'll take a look.

File: app.py [hunk 1]
```diff
@@ -1,44 +1,25 @@
-import datetime
-import hashlib
-
-import discord
+from lacus.cogs.time import TimeCog
```
Comment (pk400) [inline app.py:1]: Just a comment, but I was thinking of adding something like [isort](https://github.com/PyCQA/isort) which could help organize these imports.

File: lacus/cogs/time.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
+import discord
+from discord.ext import commands
+import datetime
+import pytz
+
+class TimeCog(commands.Cog):
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+        self.channel = None
```
Comment (pk400) [inline lacus/cogs/time.py]: Are we using `channel` for anything? We can probably get rid of it.

File: lacus/cogs/time.py [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+        await ctx.send(reply_message)
+        
+    @time.group()
+    async def add_timezone(self, ctx:commands.Context, timezone):
+        if (timezone in self.timezones):
+            await ctx.send("Timezone has already been added!")
+            return
+        
+        if (timezone not in pytz.all_timezones):
+            await ctx.send("Please enter a valid timezone name.\n A list of timezonen names can be found here: https://en.wikipedia.org/wiki/List_of_tz_database_time_zones ")
```
Comment (pk400) [inline lacus/cogs/time.py]:
```suggestion
            await ctx.send("Please enter a valid timezone name.\n A list of timezones names can be found here: https://en.wikipedia.org/wiki/List_of_tz_database_time_zones ")
```

**Review by pk400 — CHANGES_REQUESTED** [review]
Re: the git history issue, I think its okay. I created another [branch](https://github.com/Midnight-Cafe/lacus/compare/fix-git-history?expand=1) based off yours and checked the files changed and it looks fine even though I changed nothing. It must be something related to GitHub. I'll ignore the unrelated files for this though.

Everything looks good on my end, just a few comments. I think we can merge it in afterwards! :heavy_check_mark:

**Comment by jyorien** [comment]
I've changed the typo and removed the channel

**Review by pk400 — APPROVED** [review]
Wooo! :tada: Looks good to me!

**Pull request merged**